tests/hopp/test_wind.py

```diff
@@ -129,56 +138,91 @@ def test_changing_system_capacity_pysam(site):
         model.system_capacity_by_rating(n)
         assert model.system_capacity_kw == approx(n)
 
+
 #################### FLORIS tests ################
 def test_floris_num_turbines(site):
-    floris_config_path = (
-        ROOT_DIR.parent / "tests" / "hopp" / "inputs" / "floris_config.yaml"
-    )
+    floris_config_path = (ROOT_DIR.parent / "tests" / "hopp" / "inputs" / "floris_config.yaml")
     f_config = load_yaml(floris_config_path)
     floris_n_turbines = len(f_config["farm"]["layout_x"])
-    config = WindConfig.from_dict({'num_turbines': 20, "turbine_rating_kw": 5000, "model_name": "floris", "timestep": [1, 8760], "floris_config": floris_config_path})
-    with pytest.raises(UserWarning) as err:
-        model = WindPlant(site, config=config)
-    assert str(err.value) == f"num_turbines input ({config.num_turbines}) does not equal number of turbines in floris layout ({floris_n_turbines})"
-   
+    config = WindConfig.from_dict(
+        {
+            'num_turbines': 16,
+            "turbine_rating_kw": 5000,
+            "model_name": "floris",
+            "timestep": [1, 8760],
+            "floris_config": floris_config_path
+        }
+    )
+    model = WindPlant(site, config=config)
+    xcoords, ycoords = model._system_model.wind_farm_layout
+    assert len(xcoords) == config.num_turbines
+    assert len(ycoords) == model.num_turbines
+    assert model._system_model.nTurbs == model.num_turbines
+    # with pytest.raises(UserWarning) as err:
+    # assert str(err.value) == f"num_turbines input ({config.num_turbines}) does not equal number of turbines in floris layout ({floris_n_turbines})"
```

Comment:
Marking this as a future issue, to add a test to make sure that checks that this error is raised.